src/components/common/select/Select.tsx

```diff
@@ -39,7 +41,11 @@ export const Select = <TValue, >({
                      }
                  }}>
         <span
-            style={selectedValue !== null && selectedValue !== undefined ? {color: "#061125"} : {color: "#61615C"}}> {selectedValue !== null && selectedValue !== undefined ? selectedValue.toString() : 'Статус'}</span>
+            className={classNames('empty', {
+                'not-empty': selectedValue !== null && selectedValue !== undefined})}
```

Comment:
One other suggestion
If you would track span state using classes `empty/not-empty` then it will be confuthin seens those two classes together like `<span class="empty not-empty">`

It would not be clear if span `is empty or not?`

you can do next
```
const isSelectedValuePresent = !!selectedValue;

<span
  className{classNames({
     'empty': !isSelectedValuePresent,
      'not-empty': isSelectedValuePresent
  })}
>
```

in that way you would have only one class present for span and it would be good for tracking span state